src/adv_diff/AdvDiffSemiImplicitHierarchyIntegrator.cpp

```diff
@@ -908,23 +908,37 @@ AdvDiffSemiImplicitHierarchyIntegrator::postprocessIntegrateHierarchy(const doub
 /////////////////////////////// PROTECTED ////////////////////////////////////
 
 void
-AdvDiffSemiImplicitHierarchyIntegrator::resetHierarchyConfigurationSpecialized(
-    const Pointer<BasePatchHierarchy<NDIM> > base_hierarchy,
-    const int coarsest_level,
-    const int finest_level)
+AdvDiffSemiImplicitHierarchyIntegrator::regridHierarchyBeginSpecialized()
 {
-    const Pointer<BasePatchHierarchy<NDIM> > hierarchy = base_hierarchy;
-    const int finest_hier_level = hierarchy->getFinestLevelNumber();
-    d_hier_fc_data_ops->setPatchHierarchy(hierarchy);
-    d_hier_fc_data_ops->resetLevels(0, finest_hier_level);
     for (const auto& Q_var : d_Q_var)
     {
         d_Q_convective_op_needs_init[Q_var] = true;
     }
+    AdvDiffHierarchyIntegrator::regridHierarchyBeginSpecialized();
+    return;
+} // regridHierarchyBeginSpecialized
+
+void
+AdvDiffSemiImplicitHierarchyIntegrator::resetHierarchyConfigurationSpecialized(
```

Comment:
Same as above: Can you remove this specialization since it only calls the parent class implementation?

Reply:
This one looks like it could go away though?

Reply:
> Same as above: Can you remove this specialization since it only calls the parent class implementation?

Done 👍